futurenhs.automation/wdio.conf.js

```diff
@@ -41,9 +41,9 @@ exports.config = {
     ],
     suites: {
         fullRegression: [
-            './features/**/platformAdmin.feature',
             './features/**/userAccess.feature',
             './features/**/search.feature',
+            './features/**/platformAdmin.feature',
```

Comment:
Will be useful at some stage to do a further run on features to see which run fast and to organise them accordingly when possible.

Reply:
Yes, currently they're organised in such a way for how data is manipulated as tests progress. However I want to try and do another review of all data and see about where this issue can be minimised even more so.